hbase-common/src/main/java/org/apache/hadoop/hbase/TableName.java

```diff
@@ -260,6 +260,10 @@ public String getQualifierAsString() {
     return qualifierAsString;
   }
 
+  public String getMetricPrefixTableName() {
```

Comment:
TableName is IA.Public so we'd better not add new method unless you are sure it is useful for everyone.